### DISCUSSION TITLE
Wechselrichter hat tagsüber zwischenzeitlich keine Produktion

### DISCUSSION THREAD

**Discussion by 2000ede** [tbnobody/OpenDTU#1799] [Q&A]
Hallo,

in einem anderen Ticket versuche ich ja meine Instabilitäten der DTU herauszufinden/zu mildern. Ich habe mir eine Überwachung der Wechselrichter gebastelt bzw hobby-programmiert.

Neben der sporalischen Instabilität der DTU (aktuell läuft es erstaunlich ruhig und gut) erkenne ich über die Überwachung 3-4 mal Wechselrichter-Produktionsausfälle: Die DTU (mit drei WR) läuft aktuell sauber (bzw doch langsamer aber ohne Absturz), die DTU hält sauber den Kontakt zum Wechselrichter (Sekundenzähler), aber die aktuelle Erzeugung (w) ist auf null. Es dauert bis zu 30 Minuten, dann schafft es der WR selber, wieder in Produktion zu kommen.

Kennt jemand diese Ausfälle? Und hat ggf. Tipps? Danke!

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/7c090796-10be-4c37-a2b4-284c32a9fd9e)

Nur Merker / Interesshalber: Das andere Ticket: (https://github.com/tbnobody/OpenDTU/discussions/1788).

**Comment by 2000ede** [comment]
Ergänzung: 
- es liegt nicht an schwacher Sonne
- nach Neustart ist er gleich auf 130 Watt
- DTU nimmt kein neues Limit für den WR
- json DTU-Abruf über browser geht flüssig

**Comment by 2000ede** [comment]
Hier ein anderes Beispiel: Hier ist es anders rum: Produktion ist da, aber Limit springt auf 0 und bleibt da so lange man es nicht per Hand wieder in der DTU hochsetzt. Bei diesem Wechselrichter erfolgt nur ein Logging, er bekommt keine Limite von Außen (Fest auf ein Limit eingestellt)

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/c145aaa4-0470-4934-9c03-77e2f995c1a3)

![grafik](https://github.com/tbnobody/OpenDTU/assets/37264959/902815dc-52ef-46f0-a27d-0e61d976b78f)

Edit: nach 4 Minuten hat er sich diesmal von alleine erholt und das Limit war wieder da

**Comment by Omega13x** [comment]
Falsch konfiguriert. Du musst der Regelung klar machen nicht auf Null runterzuregeln. Wenn der/die WR den Befehl NULL bekommen, dann schalten sie sich aus. Dann dauert es zw. 4 und 15 Minuten, bis sich der/die WR wieder einschalten. Deswegen ist bei dir auch OpenDTU angeblich instabil.

**Reply by 2000ede** [reply]
Das ist so nicht der Fall, im Gegenteil. Bei Sonne und Strombedarf passiert es.
Niedrigstes Limit sind 8% wegen Grundlast. Und die fährt er auch bis auf die seltenen Ausnahmen auch gut.

**Comment by Omega13x** [comment]
"Time calibration", heißt das du noch mit einer zweiten DTU auf den/die Wechselrichter zugreifst. Das wäre dann auch noch eine Erklärung für deine Probleme.

**Reply by 2000ede** [reply]
nein, eine zweite DTU war definitiv nicht im Spiel
